Updated dependencies jwiki and JSON-java to their latest versions.

fastily opened the pull request:
Also included: Added newest version of FGUI, light refactor for
sanitycheck() in GRThread, moved source files to a more appropriate
package name (globalreplace), junked files which are not used in
globalreaplce, and updated travis-ci/ant build file.

MarcoFalke commented:
Code looks good. Please also assign a new `VERSION_NUM`.

On another note: What is the easiest way to verify the `.jar` files in fact contain the correct content? Also git is not meant for binary files. Maybe fetching them externally could make sense. Or using a submodule?

fastily commented:
I bumped the minor version, since this is just a small bug fix to end users.

The jwiki-v1.1.jar has an md5 of `fc1d56f6cbcc1b1a4aae8b5769b0d08b`.  I have updated the project's [releases](https://github.com/fastily/jwiki/releases) page to reflect this.  

I agree that git isn't ideal for binaries, however our project is very small and only has one (jwiki) dependency.  I'd like to get jwiki on maven for its next release, so whenever that happens I can configure maven dependencies if you all want that.  I will say that although it is convenient, it does introduce additional overhead, and it will make it much harder to contribute for newer programmers who are unfamiliar with java/maven.

I'm against using submodules.  You cannot submodule specific folders, you need to explicitly (recursively) clone submodules which is confusing to collaborators, and it is way too easy to accidentally break parts of your local git repo.

MarcoFalke commented:
Ok, looks good.

Could you also squash the commit "removed redundant JSON-java.jar" into the commit which added the JSON-java.jar?

Commit:
Updated dependeny jwiki to its latest version.

Also included:
*Added newest version of FGUI, light refactor for
sanitycheck() in GRThread, moved source files to a more appropriate
package name (globalreplace), junked files which are not used in
globalreaplce, and updated travis-ci/ant build file
* Bump version number per request, removed redundant JSON-java.jar
* light re-factor in GlobalReplace to take advantage of
lambdas

fastily commented:
@MarcoFalke: Done.

Steinsplitter commented:
:+1: )

MarcoFalke commented:
Great work!

Steinsplitter commented:
@MarcoFalke  Can you create a new https://github.com/Commonists/GlobalReplace/releases please (for easy download?
